src/lightning/pytorch/trainer/setup.py

```diff
@@ -167,17 +162,21 @@ def _log_device_info(trainer: "pl.Trainer") -> None:
         from lightning_graphcore import IPUAccelerator
 
         num_ipus = trainer.num_devices if isinstance(trainer.accelerator, IPUAccelerator) else 0
+        available = IPUAccelerator.is_available()
     else:
         num_ipus = 0
-    rank_zero_info(f"IPU available: {_IPU_AVAILABLE}, using: {num_ipus} IPUs")
+        available = False
+    rank_zero_info(f"IPU available: {available}, using: {num_ipus} IPUs")
```

Comment:
you can still have IPU available even if yu you do not install lightning-graphcore

Reply:
IPUs are supported through `lightning-graphcore`. For the Trainer, IPUs are effectively unavailable without this package.

All acceleratorss' availability is checked via `Accelerator.is_available()`